drivers/video/video_ctrls.c

```diff
@@ -549,30 +549,31 @@ static inline const char *video_get_ctrl_name(uint32_t id)
 	}
 }
 
-int video_query_ctrl(const struct device *dev, struct video_ctrl_query *cq)
+int video_query_ctrl(struct video_ctrl_query *cq)
 {
 	int ret;
 	struct video_device *vdev;
 	struct video_ctrl *ctrl = NULL;
 
-	__ASSERT_NO_MSG(dev != NULL);
```

Comment:
[non-blocking]
If wanting to preserve the assert, this could become:
```suggestion
	__ASSERT_NO_MSG(cq->dev != NULL);
```

Reply:
It is checked in `video_find_vdev()` but yes, better to assert here. Done.